### PR TITLE
Readme bővítés és minimális kód változtatások

### PR DESCRIPTION
**Pull request by skornel02**

### CODE REVIEW

File: Net6APIBasicAuthApiKey/Auth/AuthenticationHandler.cs [hunk 1]
```diff
@@ -19,29 +20,26 @@ public class AuthenticationHandler : AuthenticationHandler<BasicAuthenticationOp
...
+            Response.Headers.Add(HeaderNames.WWWAuthenticate, BasicAuthenticationSchemeName);
+            var result = AuthenticateResult.Fail("Proper authentication information is necessary to use this app (missing header)!");
             return Task.FromResult(result);
         }
 
         if (Options.ServiceAccessInfo is null)
         {
-            base.Logger.LogError("Error in Authentiacation middleware. {ServiceAccessInfo} is null",
-                nameof(Options.ServiceAccessInfo));
+            Logger.LogError("Error in Authentication middleware. {} is null", nameof(Options.ServiceAccessInfo));
```
Comment (spori-bs) [inline Net6APIBasicAuthApiKey/Auth/AuthenticationHandler.cs]: ServiceAccessInfo is missing "{}"

File: Net6APIBasicAuthApiKey/Auth/AuthorizationService.cs [hunk 2]
```diff
@@ -17,34 +17,32 @@ public AuthorizationService(ILogger<AuthorizationService> logger, IOptions<Servi
...
+            {
+                _logger.LogInformation("Missing API-KEY:{}", ip);
+                return Task.FromResult(AuthorizationResult.Failed(AuthorizationFailure.Failed(requirements)));
             }
-            else
+
+            var headerApiKey = httpContext.Request.Headers[ApiKeyHeaderValue];
+            if (_serviceAccessInfo.ApiKey != headerApiKey)
             {
+                _logger.LogInformation("Invalid API-KEY:{}:{}", headerApiKey, ip);
```
Comment (spori-bs) [inline Net6APIBasicAuthApiKey/Auth/AuthorizationService.cs]: Format string power.. I like the {0} {1} ... or {ip} style.. much readable when you use multiple params..
The empty brackets is avoidable for me

**Review by spori-bs — CHANGES_REQUESTED** [review]
You should change the formatstrings style.
When you use a method formatstring and "params object[]" 
consider using interpolated strings. Sometimes you can get better results. (Boxing into array)

File: Net6APIBasicAuthApiKey/Auth/AuthorizationService.cs (hunk 2 above)
Comment (skornel02) [reply Net6APIBasicAuthApiKey/Auth/AuthorizationService.cs]: I think that the {ip} style is the more descriptive.

**Comment by skornel02** [comment]
> You should change the formatstrings style. When you use a method formatstring and "params object[]" consider using interpolated strings. Sometimes you can get better results. (Boxing into array)

I would generally prefer using interpolated strings, but when it comes to logging (to the best of my knowledge) using formatted strings is the de facto standard. 
Object boxing is a performance burden on the runtime, but a logger can simply not call the format method *(if it is not required)* and this is a stark contrast to the interpolated string that is ALWAYS evaluated.

**Review by spori-bs — APPROVED** [review]

**Pull request merged**